installing/installing_openstack/installing-openstack-installer-ovs-dpdk.adoc

```diff
@@ -63,20 +63,21 @@ The cluster is installed and prepared for configuration. You must now perform th
 
 include::modules/cluster-telemetry.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
 .Additional resources
 
 * See xref:../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring] for more information about the Telemetry service
 
+[role="_additional-resources"]
 [id="additional-resources_installing-openstack-installer-ovs-dpdk"]
 == Additional resources
-* See xref:../../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#cnf-understanding-low-latency_cnf-master[Performance Addon Operator for low latency nodes] for information about configuring your deployment for real-time running and low latency.
+* xref:../../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#cnf-understanding-low-latency_cnf-master[Low latency tuning of OpenShift Container Platform nodes]
 
 [id="next-steps_installing-openstack-installer-ovs-dpdk"]
 == Next steps
 
 * To complete OVS-DPDK configuration for your cluster:
```

Comment:
Would move this into the item it used to introduce since there is now only one...
```suggestion
```